README.md

```diff
@@ -0,0 +1,103 @@
+## Introduction
+This is a demo module for Islandora CLAW. It illustrates the modeling of a content type and creation of a drupal module for CLAW.
+
+
+## Steps to create claw_porcus
+### 1 - Create a new Media bundle
+* Go to `/admin/structure/media/add`
+* Provide a label to the bundle: PORCUS_TEXT
+* Lave the type provider as Generic media
```

Comment:
Just suggestion. Data types, drupal provided choices, file names could go in some other notation like in between `label` or _italic_. Example "Generic media" would become "`Generic media`". But not sure what is best really so just a comment